CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 Entries are listed in reverse chronological order.
 
-## Unreleased
+## 0.4.1
 
-* Migrate to `group` 0.13, `jubjub` 0.10, `pasta_curves` 0.5
+* Add Pallas and Jubjub ciphersuites and FROST support (#33)
+* Migrate to `group` 0.13, `jubjub` 0.10, `pasta_curves` 0.5 (#44)
 
 ## 0.4.0
 
+* MSRV is now 1.60.0 (note: this was noticed after the crate was released)
```

Comment:
Now that the MSRV is recent enough to support it, we should add a `rust-version = "1.60"` line underneath the `edition` line in `Cargo.toml`.

Reply:
Done in 69083f5